emoji/src/main/java/com/vanniktech/emoji/EmojiEditText.java

```diff
@@ -82,12 +95,12 @@ public final void setEmojiSize(@Px final int pixels, final boolean shouldInvalid
   }
 
   /** sets the emoji size in pixels with the provided resource and automatically invalidates the text and renders it with the new size */
-  public final void setEmojiSizeRes(@DimenRes final int res) {
+  @Override public final void setEmojiSizeRes(@DimenRes final int res) {
     setEmojiSizeRes(res, true);
   }
 
   /** sets the emoji size in pixels with the provided resource and invalidates the text and renders it with the new size when {@code shouldInvalidate} is true */
```

Comment:
Can you move all of the javadoc from this up in the interface? and then remove it from the implemenations?